c2corg_api/views/waypoint.py

```diff
@@ -28,7 +29,11 @@
 from functools import lru_cache
 from sqlalchemy.orm import joinedload, load_only
 from sqlalchemy.orm.util import aliased
-from sqlalchemy.sql.expression import and_
+from sqlalchemy.sql.elements import literal_column
+from sqlalchemy.sql.expression import and_, union
+
+# the number of routes that are included for waypoints
+NUM_ROUTES = 30
```

Comment:
> It is out of question to show 100 routes for a waypoint.

?
There are several such examples in v5 :)
We'll have to explain the client why this can't be done in v6.

You don't answer my question about how to choose the route to eventually display in the WP page.
I think this is not trivial.

I agree that having too many routes in a WP most likely means the WP definition is questionable (for instance "massif des Calanques") and that it should be changed/splitted. But I'm not sure that's the association thinks so...